en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -570,11 +570,10 @@ This also works for pagination at the Controller level::
         }
     ];
 
-.. note::
+.. warning::
 
-    When you limit the fields that are fetched from an association, you **must**
-    ensure that the foreign key columns are selected. Failing to select foreign
-    key fields will cause associated data to not be present in the final result.
+    If the results are missing association entities, make sure the foreign key columns
+    are selected in the query.  Without the foreign keys, the ORM cannot find matching rows.
```

Comment:
Is there an issue with your IDE? The text you added contains double space after sentences for some reason.

Reply:
I think I'm just typing like that sometimes without noticing.  I'm not sure why - maybe I used to write papers like that.  I'm doing it now apparently lol.